soc/nxp/layerscape/ls1046a/Kconfig.soc

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_LS1046A
+	bool
+	select SOC_FAMILY_LAYERSCAPE
+
+config SOC_LS1046A
+	bool
+	select SOC_SERIES_LS1046A
+	help
+		NXP Layerscape ls1046a
```

Comment:
help text indentation is 1 tab followed by 2 spaces

Reply:
Fixed, thanks.